src/drivers/JwtDriver.ts

```diff
@@ -84,7 +82,6 @@ export default class JwtDriver implements AuthDriverInterface {
       ) {
         this.authService.refresh().then(
           () => {
-            token = JSON.parse(this.storageService.get(this.storagePath));
```

Comment:
We need this line, after you refresh, your token has changed , and you need to retrieve it

Reply:
This should use your new function to call